### PR TITLE
Added improvements for RecentChapters. Closes #320

### LINKED ISSUE #320: More options to Recent Updates tab
It would be nice to refresh mangas, download/delete/mark as read/unread multiple chapters in recent updates tab. Same or similar to opening individual manga as in screenshot.
screenshot_2016-05-31-22-48-19_eu kanade tachiyomi
Thank you for your hard work

### PR DESCRIPTION
**Pull request by NoodleMage**
screenshot_20160602-200002_framed
screenshot_20160602-200033_framed

#320

### CODE REVIEW

File: app/src/main/java/eu/kanade/tachiyomi/ui/recent/RecentChaptersFragment.kt [hunk 1]
```diff
@@ -70,60 +123,108 @@ class RecentChaptersFragment : BaseRxFragment<RecentChaptersPresenter>(), Flexib
...
         // Update toolbar text
         setToolbarTitle(R.string.label_recent_updates)
     }
 
     /**
+     * Returns selected chapters
+     * @return list of [MangaChapter]s
+     */
+    fun getSelectedChapters(): List<MangaChapter> {
+        return adapter.selectedItems.map { adapter.getItem(it) as MangaChapter }.toList()
```
Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/ui/recent/RecentChaptersFragment.kt]: `return adapter.selectedItems.map { adapter.getItem(it) }.filterIsInstance(MangaChapter::clas.java).toList()`

Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/ui/recent/RecentChaptersFragment.kt]: `return adapter.selectedItems.map { adapter.getItem(it) as? MangaChapter }.filterNotNull()`

Another error free approach

File: app/src/main/java/eu/kanade/tachiyomi/ui/recent/RecentChaptersPresenter.kt [hunk 2]
```diff
@@ -315,4 +346,8 @@ class RecentChaptersPresenter : BasePresenter<RecentChaptersFragment>() {
         chapter.status = Download.NOT_DOWNLOADED
     }
 
+    fun fetchChaptersFromSource() {
```
Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/ui/recent/RecentChaptersPresenter.kt]: This (and the swipe refresh) shouldn't be needed. The adapter is updated automatically whenever something changes. Unless I misunderstood something...

Comment (NoodleMage) [inline app/src/main/java/eu/kanade/tachiyomi/ui/recent/RecentChaptersPresenter.kt]: Not sure. Doesn't the restartabletable unsubscribe after updating? This means that the recent won't get updated until the presenter is created again. I'm not certain about this though.

Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/ui/recent/RecentChaptersPresenter.kt]: Nope, latest cache is subscribed until the presenter is destroyed.

Comment (NoodleMage) [inline app/src/main/java/eu/kanade/tachiyomi/ui/recent/RecentChaptersPresenter.kt]: I see. Yeah then it's absolutely pointless xD. I can remove it in an hour or so. Or can you do it? ;)

Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/ui/recent/RecentChaptersPresenter.kt]: I will, don't worry.

**Pull request merged**